lib/ain-cpp-imports/src/bridge.rs

```diff
@@ -29,17 +29,17 @@ pub mod ffi {
 
         fn getChainId() -> u64;
         fn isMining() -> bool;
-        fn publishEthTransaction(data: Vec<u8>) -> String;
+        fn publishEvmTransaction(data: Vec<u8>) -> String;
         fn getAccounts() -> Vec<String>;
         fn getDatadir() -> String;
         fn getNetwork() -> String;
-        fn getEthMaxConnections() -> u32;
+        fn getEvmMaxConnections() -> u32;
```

Comment:
Three is no EVM Connection. The connections are for ETH-JSON RPC. And as such I'd argue the original naming is technically more accurate.